### ISSUE TITLE
Remove second travel time range slider

### ISSUE THREAD

**Issue by abyrd** [conveyal/analyst-server#178]
We currently have two slider handles, presumably for calculating accessibility "donuts". I'm not sure anyone will use this, and if they do it's a very advanced feature. This is UI clutter that should be removed, at least by default.
